.github/workflows/artifacts.yml

```diff
@@ -0,0 +1,113 @@
+name: Upload Release Artifacts
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag'
+        required: true
```

Comment:
Right now I have this as a manually-run workflow, but we could change it to happen automatically for tags of the form `v*`, like zkgroup does.